### PR TITLE
Improve models and new DAOs

### PR DESCRIPTION
**Pull request by AndrewLiaIH**
DAOs are incomplete; will need to add a method in BaseDao for GetById queries

### CODE REVIEW

File: DAL/EmployeeDAO.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
+﻿using System.Data;
+using System.Data.SqlClient;
+using Model;
+
+namespace DAL
+{
+    public class EmployeeDAO : BaseDAO
```
Comment (AnastasiaIU) [inline DAL/EmployeeDAO.cs]: Please, follow the naming conventions for C#, e.g. BaseDao.

Naming example:
https://learn.microsoft.com/en-us/cpp/mfc/reference/cdaoworkspace-class?view=msvc-170

Comment (AnastasiaIU) [reply DAL/EmployeeDAO.cs]: Same for ID, it should be Id

**Review by AnastasiaIU — CHANGES_REQUESTED** [review]
Please, change the naming with accordance to the convetion and finish the method GetEmployeeById.

**Review by OrestPokotylenko — CHANGES_REQUESTED** [review]
The same thing what Sia said, pleace change the namings. It looks good for the rest

**Review by AnastasiaIU — APPROVED** [review]
Looks good

**Review by OrestPokotylenko — APPROVED** [review]

**Pull request merged**